app/controllers/catalog_controller.rb

```diff
@@ -85,6 +85,16 @@ class CatalogController < ApplicationController
     }
 
     config.add_facet_field 'dc_rights_s', label: 'Access', limit: 8, partial: 'icon_facet', show: false
```

Comment:
Will the dc_rights_s field have values other than "restricted" or "public" in the dc_rights_s field that you don't want to facet on? If so, this line should probably be removed.